### PR TITLE
chore: improve handling and error reporting of exported default varia…

### PR DESCRIPTION
**Pull request by jensneuse**

### CODE REVIEW

**Review by thisisnithin — APPROVED** [review]

File: router/core/operation_processor.go [hunk 1]
```diff
@@ -569,6 +575,17 @@ func (o *OperationKit) normalizeNonPersistedOperation() (cached bool, err error)
 	return false, nil
 }
 
+func (o *OperationKit) nullifyExportedVariables(variables []byte) []byte {
+	if variables == nil {
+		return nil
+	}
+	// min meaningful variables = {"a":1}
+	if len(variables) < 7 {
+		return nil
```
Comment (StarpTech) [inline router/core/operation_processor.go:584]: Why do we return nil when we have variables? Could you add more insights into the comment?

File: router/core/operation_processor.go [hunk 2]
```diff
@@ -616,6 +633,13 @@ func (o *OperationKit) loadPersistedOperationFromCache() (ok bool, err error) {
 
 // populateDefaultVariablesFromExportedDefaults iterates through the exported default variables and sets missing ones in the variables
 func (o *OperationKit) populateDefaultVariablesFromExportedDefaults(exportedVariables, override []byte) ([]byte, error) {
+	if exportedVariables == nil {
+		return override, nil
+	}
+	if len(exportedVariables) < 7 {
```
Comment (StarpTech) [inline router/core/operation_processor.go:639]: This is duplicated.

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: router/core/operation_processor.go (hunk 1 above)
Comment (jensneuse) [reply router/core/operation_processor.go:594]: added

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**